Yosemite/Yosemite/Stores/SystemStatusStore.swift

```diff
@@ -31,6 +31,8 @@ public final class SystemStatusStore: Store {
             synchronizeSystemPlugins(siteID: siteID, completionHandler: onCompletion)
         case .fetchSystemPlugin(let siteID, let systemPluginName, let onCompletion):
             fetchSystemPlugin(siteID: siteID, systemPluginName: systemPluginName, completionHandler: onCompletion)
```

Comment:
nit: do we want to share the code by calling `fetchSystemPlugin(siteID: siteID, systemPluginNameList: [systemPluginName], completionHandler: onCompletion)` here?